torch/fx/graph.py

```diff
@@ -952,6 +952,20 @@ def check_arg(arg : Node, n : Optional[Node] = None) -> None:
                             raise RuntimeError(f'Node {node} target {node.target} references nonexistent attribute '
                                                f'{atom} of {seen_qualname}')
 
+    def run_dce(self):
```

Comment:
I'd prefer a name like `eliminate_dead_code`. Not sure if our general userbase will be familiar with the DCE initialism